pkgs/applications/networking/taler/default.nix

```diff
@@ -1,57 +1,100 @@
 { lib, stdenv, fetchurl, curl, gnunet, jansson, libgcrypt, libmicrohttpd
-, qrencode, libsodium, libtool, pkg-config, postgresql, sqlite }:
+, qrencode, libsodium, libtool, pkg-config, postgresql, sqlite, autoreconfHook, jinja2, recutils, wget, jq, gettext, texinfo
```

Comment:
packages outside of pythonPackages should use python or pythonPackages as an input instead of the individual package.